components/style/cascade_info.rs

```diff
@@ -60,6 +60,35 @@ impl CascadeInfo {
         }
     }
 
+    /// Called when a boxed property is cascaded.
+    /// This should be same as `on_cascade_property`.
+    /// FIXME(canaltinova): Find a better method to handle boxed properties.
+    ///
+    /// NOTE: We can add a vast amount of information here.
+    #[inline]
+    pub fn on_cascade_boxed_property<T>(&mut self,
```

Comment:
I had to create similar function to `on_cascade_property`. It's not a mako file, I couldn't fix it with mako.